### ISSUE TITLE
A11y_SQLAzureDataStudio_DeployDacpac_DeployDacpacSettings_Screenreader: When opened the page, focus moves to 'Next' button but screenreader doesn't announce control's information

### ISSUE THREAD

**Issue by shweta255** [microsoft/azuredatastudio#9325]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudio;#A11yMAS;#A11ySev2;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#Win10;#MAS1.3.1;#Screenreader;

**Environment Details:**
Version: 1.16.0-insider
Commit: 506c6a5
Date: 2020-02-17T05:43:18.399Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363
Additional Details:
MAS References: MAS1.3.1
NVDA Version: 2019.2
JAWS Version: 2019.1912.1 ILM

**Repro Steps:**
Launch Azure Data Studio application.
Connect to server with windows credentials.
Navigate to Extensions from left menu.
Search & install Dacpac extension.
Navigate to Connections from left menu.
Expand the server connected in step 2.
Right click an on prem database and select Data-tier application wizard from Context menu.
Select first radio button.
Navigate till next button & hit enter.
Launch NVDA screenreader
Navigate to column headers 'Settings' and 'value' using down arrow key

**Same issue is observed  with JAWS Screenreader.** 

**Same issue is observed in Extract Dacpac Settings screen:** Navigation:
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Dacpac extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Data-tier application wizard from Context menu.
8. Select second radio button.
9. Navigate till next button & hit enter.

**Actual:**
While screenreader is on and opened the DeployDacpacSettings screen, focus moves to 'Next' button but screenreader doesn't announce control's information.

**Expected:**
While screenreader is on and opened the DeployDacpacSettings screen, Screenreader should announce control's information of 'Next' button as focus is at 'Next' button.

**User Impact:**
Screen reader users will face difficulty to understand current focus on the screen.

**Attachment link for Reference:**
[9325_A11y_SQLAzureDataStudio_DeployDacpac_DeployDacpacSettings_Screenreader.zip](https://github.com/microsoft/azuredatastudio/files/4274804/9325_A11y_SQLAzureDataStudio_DeployDacpac_DeployDacpacSettings_Screenreader.zip)

**Comment by alanrenmsft** [comment]
the focus stayed on 'Next' button, i don't think it should be re-announced. i am closing the issue as by design. please reopen if you don't agree with me. @shweta255

**Comment by BhanuPrakashGit** [comment]
GithubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
